Add guest content for command occ user:report (#3581)
owncloud/docs: modules/admin_manual/pages/configuration/server/occ_commands/core_commands/user_commands.adoc

== Generating a User Count Report

Generate a simple report that counts all users, including users on external user authentication servers such as LDAP

> and guest users for example created by the guests app.

Reviewer: ```suggestion
and guest users which are created by the guests app.
```
